src/cmds/validate.js

```diff
@@ -0,0 +1,62 @@
+const chalk = require('chalk');
+const fs = require('fs');
+const OASNormalize = require('oas-normalize');
+
+exports.command = 'validate';
+exports.usage = 'validate [file] [options]';
+exports.description = 'Validate your OpenAPI/Swagger definition.';
+exports.category = 'apis';
+exports.position = 2;
+
+exports.hiddenArgs = ['spec'];
+exports.args = [
+  {
+    name: 'spec',
+    type: String,
+    defaultOption: true,
+  },
+];
+
+exports.run = async function (opts) {
+  const { spec } = opts;
+
+  async function validateSpec(specPath) {
+    const oas = new OASNormalize(specPath, { colorizeErrors: true, enablePaths: true });
+
+    return oas
+      .validate(false)
+      .then(api => {
+        if (api.swagger) {
+          console.log(chalk.green(`${specPath} is a valid Swagger API definition!`));
+        } else {
+          console.log(chalk.green(`${specPath} is a valid OpenAPI API definition!`));
+        }
+      })
+      .catch(err => {
+        return Promise.reject(new Error(err.message));
```

Comment:
Does this return with an error exit code cos we're rejecting the promise? Via this: https://github.com/readmeio/rdme/blob/b5d613d58c8002aa1e1f22e6a4479b32bb1cbf77/bin/rdme#L26?

Reply:
@domharrington yeah all errors bubble back up to that line and die out with that exit code.